src/app/api/ai-plugin/route.ts

```diff
@@ -19,25 +19,26 @@ export async function GET() {
     },
     servers: [
       {
-        url: "https://snapshot-agent-theta.vercel.app/"
-      }
+        url: "https://snapshot-agent-theta.vercel.app/",
+      },
     ],
     "x-mb": {
       "account-id": "0x66268791B55e1F5fA585D990326519F101407257",
       email: "nate@bitte.ai",
       assistant: {
         name: "Snapshot DAO Agent",
         description:
-          "An agent that constructs EVM signature requests and validates cryptographic signatures for voting on snapshot DAOs. Use the generate-evm-tx primitive to create signature requests for transactions, personal messages, or EIP-712 typed data after eth_signTypedData is called. After a user signs a request, automatically validate the signature using the validate tool to verify authenticity and always inclue the message that was signed.",
+          "An agent that constructs EVM signature requests and validates cryptographic signatures for voting on snapshot DAOs. Use the generate-evm-tx primitive to create signature requests for transactions, personal messages, or EIP-712 typed data after eth_signTypedData is called. After a user signs a request, automatically validate the signature using the validate tool to verify authenticity and always include the stringified typedData that was signed.",
         instructions: `
           After the eth_signTypedData tool is signed You create EVM transactions and signature requests using the generate-evm-tx primitive.
-          After a message or typed data is signed, pass the message, evmAddress and signature back into the validate tool.
+          After a message or typed data is signed, pass the stringified message, evmAddress and signature back into the validate tool.
           To validate a signature, you must provide three pieces of information: 
-          1) REQUIRED: the original message/data that was signed, 
+          1) REQUIRED: the original stringified typedData that was signed, 
```

Comment:
This is fine in the case of typed data (which is here), but not for messages. I thought these specifics were better placed on the tool description.